ci/pipeline.yaml

```diff
@@ -22,6 +30,13 @@ resources:
         - ci/Dockerfile
         - ci/pipeline.yaml
 
+  - name: pull-requests
+    type: pull-request
+    icon: github
+    source:
+      repository: vmware-tanzu/asset-relocation-tool-for-kubernetes
+      access_token: ((github.access_token))
```

Comment:
From their documentation

> One thing to keep in mind however, is that pull requests that are opened from a fork and commits to said fork will not generate notifications over the webhook. So if you have a repository with little traffic and expect pull requests from forks, you'll need to discover those versions with check_every: 1m for instance. check in this resource is not a costly operation, so normally you should not have to worry about the rate limit.

we should expect people and I'll do it myself, to use forks instead of branching the main repo, in that case if I understood correctly we might need to add smth like

`check_every: 1m`

Reply:
Did you take a look at this? I am afraid that if we do not do it, we will not be able to receive upstream contributions properly?

Reply:
Cool. Done.

Reply:
Thanks!